### PR TITLE
edit html

### PR DESCRIPTION
**Pull request by ChristophObst**
tell me if the changes are useful

### CODE REVIEW

**Review by DominicWebDev — APPROVED** [review]
yep !

File: html.html [hunk 1]
```diff
@@ -0,0 +1 @@
+heute beschäftigen wir uns mit github
```
Comment (AnjaSiewert) [inline html.html:1]: I would add: "so much fun!! 😍"

**Review by AnjaSiewert — COMMENTED** [review]
LGTM 🎓
GIF

File: html.html (hunk 1 above)
Comment (DominicWebDev) [inline html.html:1]: I would add "nothing"

**Review by DominicWebDev — APPROVED** [review]

File: html.html (hunk 1 above)
Comment (Satttoshi) [inline html.html:1]: That is a html file! This should be a markdown file!

**Review by Satttoshi — COMMENTED** [review]
Beautiful

**Pull request merged**